src/main/java/net/datafaker/providers/base/Number.java

```diff
@@ -44,35 +44,45 @@ public int negative() {
 
     /**
      * @param min the lower bound (include min)
-     * @param max the lower bound (not include max)
+     * @param max the upper bound (not include max)
      * @return a random number on faker.number() between min and max
      * if min = max, return min
      */
     public int numberBetween(int min, int max) {
         if (min == max) return min;
         final int realMin = Math.min(min, max);
         final int realMax = Math.max(min, max);
-        if (realMax - realMin >= realMin && (realMin >= 0 || realMax - realMin >= realMax)) {
+        if (isValidRange(realMin, realMax)) {
             return faker.random().nextInt(realMax - realMin) + realMin;
         }
         return decimalBetween(realMin, realMax).intValue();
     }
+    private boolean isValidRange(int realMin, int realMax) {
```

Comment:
missed empty line between methods

Reply:
> missed empty line between methods

fixed, thank you for helping me with good coding practices.